.github/actions/helm-release-and-publish/action.yml

```diff
@@ -17,6 +17,12 @@ inputs:
   helm-repository-branch:
     description: 'Branch on the charts repository'
     required: false
+  helm-repository-subfolder:
+    description: 'The name of the subfolder inside the charts repository where the package should be added'
+    required: false
+  helm-repository-base-url:
+    description: 'Helm chart repo base url'
+    required: true
```

Comment:
I think it should not be required (it is not required in `helm-publish-chart` action either)

Reply:
Fixed